supertokens_python/recipe/session/recipe_implementation.py

```diff
@@ -46,10 +48,109 @@
 if TYPE_CHECKING:
     from typing import List, Union
     from supertokens_python import AppInfo
-    from supertokens_python.querier import Querier
 
-from .constants import JWKCacheMaxAgeInMs, JWKRequestCooldownInMs
 from .interfaces import SessionContainer
+from supertokens_python.querier import Querier
+
+
+class GetJWKSResult:
+    def __init__(self, keys: List[PyJWK], last_fetched: int):
+        self.keys = keys
+        self.last_fetched = last_fetched
+
+
+from typing_extensions import TypedDict
+import threading
+from os import environ
+
+
+class JWKSConfigType(TypedDict):
+    cache_max_age: int
+    refresh_rate_limit: int
+
+
+JWKSConfig: JWKSConfigType = {
+    "cache_max_age": 6000,
+    "refresh_rate_limit": 500,
+}
+
+jwks_cache: Optional[GetJWKSResult] = None
+mutex = threading.RLock()
```

Comment:
Had to define ``JWKSConfig` as a dict since if we declare seperate vars for cache_max_age and refresh_rate_limit, that won't be mutable during the tests. Dict is mutable but primitives like int, str aren't.